core/src/main/java/org/mskcc/cbio/oncokb/util/CacheUtils.java

```diff
@@ -222,6 +240,24 @@ private static void notifyOtherServices(String cmd, Set<Integer> entrezGeneIds)
             System.out.println("Register other services: " + MainUtils.getTimestampDiff(current) + " at " + MainUtils.getCurrentTime());
             current = MainUtils.getCurrentTimestamp();
 
+            final String s3AccessKey = PropertiesUtils.getProperties("aws.s3.accessKey");
+            final String s3SecretKey = PropertiesUtils.getProperties("aws.s3.secretKey");
+            final String s3Region = PropertiesUtils.getProperties("aws.s3.region");
+
+            AWSCredentials credentials = new BasicAWSCredentials(s3AccessKey, s3SecretKey);
+            AmazonS3 s3client = AmazonS3ClientBuilder.standard()
+                .withCredentials(new AWSStaticCredentialsProvider(credentials))
+                .withRegion(s3Region)
+                .build();
+            S3Object s3objectTrials = s3client.getObject("oncokb", "clinical-trials/results/trials.json");
```

Comment:
These file locations should be put on top as constants. The common folders should be shared to avoid typos.